packages/camera/camera_android/example/integration_test/camera_test.dart

```diff
@@ -205,6 +206,71 @@ void main() {
     expect(duration, lessThan(recordingTime - timePaused));
   });
 
+  testWidgets('Set description while recording', (WidgetTester tester) async {
+    final List<CameraDescription> cameras =
+        await CameraPlatform.instance.availableCameras();
+    if (cameras.length < 2) {
+      return;
+    }
+
+    final CameraController controller = CameraController(
+      cameras[0],
+      ResolutionPreset.low,
+      enableAudio: false,
+    );
+
+    await controller.initialize();
+    await controller.prepareForVideoRecording();
+
+    await controller.startVideoRecording();
+    sleep(const Duration(milliseconds: 500));
```

Comment:
1 second is a long time for a single unit test to take; why do these tests require sleeps?